Formula/pipenv.rb

```diff
@@ -71,16 +70,6 @@ def install
       PATH: "#{libexec}/tools:$PATH",
     }
     (bin/"pipenv").write_env_script(libexec/"bin/pipenv", env)
-
-    output = Utils.safe_popen_read(
-      { "PIPENV_SHELL" => "bash" }, libexec/"bin/pipenv", "--completion", { err: :err }
```

Comment:
it looks like pipenv does not has anymore the `--completion`

Reply:
```ruby
{ "_PIPENV_COMPLETE" => "bash_source" }, libexec/"bin/pipenv"
```

Reply:
This should work
```rb
output = Utils.safe_popen_read(
      { "_PIPENV_COMPLETE" => "bash_source" }, libexec/"bin/pipenv"
```

Reply:
note we can also add `fish` completions

Reply:
with the command...
```
   output = Utils.safe_popen_read( 
      { "_PIPENV_COMPLETE" => "bash_source" }, libexec/"bin/pipenv", { err: :err }
    )
    (bash_completion/"pipenv").write output
```

I am the error
```
RuntimeError: Shell completion is not supported for Bash versions older than 4.4.
Error: Failure while executing; `\{\"_PIPENV_COMPLETE\"=\>\"bash_source\"\} /usr/local/Cellar/pipenv/2021.11.9/libexec/bin/pipenv` exited with 1. Here's the output:
```

I think this command is not valid with the bash version that comes with MacOS :'(

Reply:
Let's just do ZSH and Fish then.